pdns/pdnsutil.cc

```diff
@@ -2461,6 +2461,14 @@ try
 
     return 0;
   }
+  else if (cmds[0]=="hash-password") {
+    if (cmds.size() < 2) {
+      cerr<<"Syntax: pdnsutil hash-password PASSWORD"<<endl;
```

Comment:
this would show up in the shell history, perhaps we should read the password from a prompt?

Reply:
`readpassphrase(3)` exists in the BSDs and is available in libbsd on linux.

Reply:
Adding a new dependency for that feature seems a bit too much, but why not..

Reply:
We can copy the one .c file instead, Otto pointed out.